tests/zero_conf_test.rs

```diff
@@ -22,10 +22,25 @@ mod zero_conf_test {
 
         assert_eq!(node.get_node_info().num_peers, 1);
 
+        const TEN_SATS: u64 = 10_000;
+
+        // With no channels, 10 sats invoice is to small to cover channel
+        // opening fees.
+        let invoice = node.create_invoice(TEN_SATS, "test".to_string());
+        assert_eq!(
+	    invoice.unwrap_err().to_string(),
+	    "PermanentFailure: Failed to register payment: InvalidInput: Payment amount must be bigger than fees");
+        let invoice = node.create_invoice(20_000_000, "test".to_string());
```

Comment:
`const TWENTY_K_SATS = 20_000_000` ?

Or why using a constant for one of the numbers but not the other one?